picard/formats/asf.py

```diff
@@ -138,6 +138,7 @@ class ASFFile(File):
         'djmixer': 'WM/DJMixer',
         'mixer': 'WM/Mixer',
         'artists': 'WM/ARTISTS',
```

Comment:
artists is simply a multi valued field with all the artists, artist includes things like 'feat.' etc